target/linux/x86/base-files/lib/preinit/01_sysinfo

```diff
@@ -12,12 +12,24 @@ do_sysinfo_x86() {
 
 	for file in sys_vendor board_vendor; do
 		vendor="$(cat /sys/devices/virtual/dmi/id/$file 2>/dev/null)"
+		case "$vendor" in
+		empty | \
```

Comment:
I am not a expert in shell scripts, but wouldn't it bet better to put the strings into `" "`?

Reply:
It doesn't make a difference here, as the strings to be checked against are hardcoded (don't come in indirectly via variables, potentially containing harmful characters) and are properly escaped. Using `""` gets a little finicky with `?*:` and the case matching with `to be filled by o.e.m.`, so it felt easier not to quote, but to escape here.